converter/include/k4SimDelphes/DelphesEDM4HepOutputConfiguration.h

```diff
@@ -10,81 +10,81 @@
 
 namespace k4SimDelphes {
   /**
-   * The settings steering the output of the DelphesEDM4HepConverter. Default
-   * values correspond to the ones found in delphes example cards.
-   */
+ * The settings steering the output of the DelphesEDM4HepConverter. Default
```

Comment:
Are these re-indentations really necessary? I suppose they are added automatically be by clang-format?